mautic/mautic
# Removing deprecations from CampaignBundle

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | N
| Automated tests included? | /
| Related user documentation PR URL | /
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | /
| BC breaks? | Y
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

I noticed that some deprecated methods are still present in the CampaignBundle and most of them were easy to remove. This PR removes them and replaces their usages.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Ensure campaign event processing works as before.

#### List backwards compatibility breaks:
1. See https://github.com/mautic/mautic/pull/8374/files#diff-0901ceb76939b17cd920bc69aa52d21f
